From b1c65bf7ea0bb0ba012a8f31c8e51e4ecea829e3 Mon Sep 17 00:00:00 2001
From: Naveen M K <naveen521kk@gmail.com>
Date: Tue, 12 Oct 2021 18:35:21 +0530
Subject: [PATCH 099/N] Define PY3_DLLNAME to fix build

PY3_DLLNAME is used in Python/pathconfig.c but isn't
defined. I guess it is useless but it will fix the build.

Co-authored-by: jeremyd2019 <github@jdrake.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 7cf11c5..11e3402 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4803,7 +4803,7 @@ then
 	*-*-mingw*)
 	DYNLOADFILE="dynload_win.o"
 	extra_machdep_objs="$extra_machdep_objs PC/dl_nt.o"
-	CFLAGS_NODIST="$CFLAGS_NODIST -DMS_DLL_ID='\"$VERSION\"'"
+	CFLAGS_NODIST="$CFLAGS_NODIST -DMS_DLL_ID='\"$VERSION\"' -DPY3_DLLNAME='L\"$DLLLIBRARY\"'"
 	;;
 	esac
 fi
